Makefile

```diff
@@ -32,15 +32,13 @@ depend: work
 ifndef HAS_MERCURIAL
 	pip install Mercurial
 endif
-ifndef HAS_GLIDE
-	go get -u github.com/Masterminds/glide
-endif
-ifeq ($(wildcard $(DEST)/vendor/.*),)
-		cd $(DEST) && glide install --strip-vendor
+ifndef HAS_DEP
+	go get -u github.com/golang/dep/cmd/dep
 endif
+	dep ensure
 
 depend-update: work
-	cd $(DEST) && glide update --strip-vendor
+	dep ensure -update
```

Comment:
Ahh, so for this do we in fact want to update the lock file and everything?  Kind of dangerous if so, as opposed to an update to a single package.

I thought this was just a "make sure everything is up to date" check in which case wouldn't just `dep ensure` be the right call?

Reply:
OK, it was my understanding that `make depend` installs dependencies and `make depend-update` updates those dependencies (which might also include updates to the lock file).
But I might be wrong here... Lets wait for comments from others.